cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -114,6 +114,40 @@ def test_xlint_fails(self):
         err_cnt = perform_xlint(TEST_DATA_DIR, ['toy'])
         self.assertGreater(err_cnt, 0)
 
+    def test_asset_with_invalid_displayname(self):
+        '''
+        This test verifies that when an asset has '/' in its displayname they are converted to underscores `_`
+        '''
+        content_store = contentstore()
+        course_assets = set([u'_invalid_displayname_subs-esLhHcdKGWvKs.srt'], )
+
+        import_course_from_xml(
+            self.store,
+            self.user.id,
+            TEST_DATA_DIR,
+            ['course_with_invalid_displayname'],
```

Comment:
It came to my mind as well and i tried it using the existing course, but scenario demands a course to be imported using `import_course_from_xml`. During import the asset file is verified for the displaynames and rename the displayname if it has `/`.